.github/workflows/add-release-reviewers.yml

```diff
@@ -0,0 +1,28 @@
+name: Add reviewers to release PRs
+
+on:
+  pull_request_target:
+    branches: ['master', 'next']
+    types: ['labeled']
+
+permissions: {}
+
+jobs:
+  add-reviewers:
+    if: ${{ github.event.label.name == 'release' }}
+    # Tests that label is added on the PR
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - id: get-members
+        uses: garnertb/get-team-members@v1
```

Comment:
Should we the commit hash here instead of version? Might be a bit more secure, since we are providing it with our github token